src/Codeception/Module/ZF2.php

```diff
@@ -107,8 +120,28 @@ public function _after(TestCase $test)
     }
 
     public function _getEntityManager()
-    {
-        $serviceLocator = Application::init($this->applicationConfig)->getServiceManager();
-        return $serviceLocator->get('Doctrine\ORM\EntityManager');
+    {   
+        return $this->grabServiceFromContainer('Doctrine\ORM\EntityManager');
```

Comment:
It was defined before, reason is in Doctrine2 Module, line 102:
$this->em = $this->dependentModule->_getEntityManager();
I just changed function contents